velox/core/QueryConfig.h

```diff
@@ -396,6 +396,11 @@ class QueryConfig {
   static constexpr const char* kSparkJsonIgnoreNullFields =
       "spark.json_ignore_null_fields";
 
+  /// The number of table scan operators per task. When set, it will override
+  /// the max driver count limit.
+  static constexpr const char* kTaskTableScanDriverCount =
```

Comment:
@kewang1024 : Thanks for this code.

Don't follow the reason for this config ? How does the engine determine what value to use here ?